### PR TITLE
Completing initiatives page

### PR DESCRIPTION
**Pull request by anshulkgupta**
Minor changes to team page

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
 mail = Mail(app)
 data = {}
 
-from config import SECRET_KEY
```
Comment (jackiehluo) [inline app.py]: Why'd you delete this import? The contact form shouldn't work without it!

Comment (anshulkgupta) [inline app.py]: I don't have the SECRET_KEY!

------------
*Anshul Gupta*
[M]: 646-623-6364
www.anshulkgupta.com

Comment (jackiehluo) [inline app.py]: Oh, it's in the latest config file on Slack. :+1:

Comment (anshulkgupta) [inline app.py]: should be resolved now!

**Comment by anshulkgupta** [comment]
the animations commit will resolve #16 (simple jQuery animation)

**Pull request merged**